addons/escoria-core/game/core-scripts/esc/esc_globals_manager.gd

```diff
@@ -5,30 +5,15 @@ extends Resource
 class_name ESCGlobalsManager
 
 
-const GLOBAL_ANIMATION_RESOURCES = "ANIMATION_RESOURCES"
-
-
 # Emitted when a global is changed
 signal global_changed(global, old_value, new_value)
 
 
-# A list of reserved globals which can not be overridden
-const RESERVED_GLOBALS = [
-	"ESC_LAST_SCENE",		# Contains the global_id of previous room
-	"FORCE_LAST_SCENE_NULL",	# If true, ESC_LAST_SCENE is not considered for 
-							# automatic transitions
-	"ANIMATION_RESOURCES"
-]
-
-
 # The globals registry
 export(Dictionary) var _globals = {}
 
 
-
-func _init():
-	set_global("ESC_LAST_SCENE", "", true)
-	set_global("FORCE_LAST_SCENE_NULL", false, true)
+var _reserved_globals: Dictionary = {}
```

Comment:
Please add a comment describing the attribute.

Reply:
Will do.